### ISSUE TITLE
MicroWebCli, line 260. SOCKS5 support

### ISSUE THREAD

**Issue by mansour1991** [jczic/MicroWebCli#2]
Hello,

The BND.ADDR and The BND.PORT that is returned from the proxy server is not used on line 260.

`                        cli.read(l + 2)`

According to rfc1928 on top of page 6, these are the server bound address and the server bound port.

thanks

**Comment by jczic** [comment]
Yes I know that but I really added the sock5 part quickly for a little compatibility on esp32 on a need 👎

**Comment by uxhamby** [comment]
Is there any documentation on the use of  'wCli.Socks5Addr'  ? Thanks.
